tests/pytest.ini

```diff
@@ -13,6 +13,6 @@ markers:
     platform: specify which platform testcase can be executed on: (physical, virtual, broadcom, mellanox, etc)
     supported_completeness_level: test supported levels of completeness (coverage) level (Debug, Basic, Confident, Thorough)
 
-log_cli_format: %(asctime)s %(levelname)s %(filename)s:%(funcName)s:%(lineno)d: %(message)s
-log_file_format: %(asctime)s %(levelname)s %(filename)s:%(funcName)s:%(lineno)d: %(message)s
+log_cli_format: %(asctime)s %(module)-16.16s L%(lineno)-.4d %(levelname)-5.5s| %(message)s
+log_file_format: %(asctime)s %(module)-16.16s L%(lineno)-.4d %(levelname)-5.5s| %(message)s
```

Comment:
I suggest keeping ```funcname``` in log. It would help to do a quick debug.

Reply:
Updated, pls check.